workspaces/vscode-python/Dockerfile

```diff
@@ -0,0 +1,109 @@
+FROM codercom/code-server:4.20.1-bookworm
+
+# On PL, we want to standardize on using 1001:1001 for the user. We change the
+# "coder" account's UID and GID here so that fixuid has no work to do later.
+# This speeds up container loading time drastically and avoids timeouts.
+USER root
+RUN export OLD_UID=$(id -u coder) && \
+	export OLD_GID=$(id -g coder) && \
+	export NEW_UID=1001 && \
+	export NEW_GID=1001 && \
+	groupmod -g 1001 coder && \
+	usermod -u 1001 -g 1001 coder && \
+	find /home -user $OLD_UID -execdir chown -h $NEW_UID {} + && \
+	find /home -group $OLD_GID -execdir chgrp -h $NEW_GID {} + && \
+	unset OLD_UID OLD_GID NEW_UID NEW_GID
+	
+# Install Python and required packages via miniconda
+USER coder
+COPY requirements.txt /
+RUN export arch=`uname -m` \
+ && curl -sfLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-${arch}.sh \
+ && /bin/bash Miniconda3-latest-Linux-${arch}.sh -b -p /home/coder/miniconda \
+ && PATH="/home/coder/miniconda/bin:$PATH" \
+ && /home/coder/miniconda/condabin/conda init bash \
+ && conda config --add channels conda-forge \
+ && conda run -n base pip install -r /requirements.txt
+
+# Cleaning up the cache in the same step when we install is important if you
+# want to minimize the image size.
+USER coder
+RUN ( code-server --disable-telemetry --force \
+		# vscode support for python, including debugger 
+		--install-extension ms-python.python \
+		# auto-generate templates for function docstrings 
+		--install-extension njpwerner.autodocstring \
+		# auto-fix indentation for multiline contexts 
+		--install-extension KevinRose.vsc-python-indent \
+		# visualize indentation with color 
+        --install-extension oderwat.indent-rainbow \
+		# visualize csv files in vscode window
+		--install-extension mechatroner.rainbow-csv \
+		# linting for markdown files
+		--install-extension DavidAnson.vscode-markdownlint \
+		# format tables in markdown files
+        --install-extension fcrespo82.markdown-table-formatter \
+	) && \
+	rm -rf /home/coder/.local/share/code-server/CachedExtensionVSIXs
+
+# Prepare the entrypoints. The entrypoint.sh part is based on what code-server
+# originally had, so it may need to be updated when code-server is updated. We
```

Comment:
Can you link to the original file to make future maintenance easier? https://github.com/coder/code-server/blob/main/ci/release-image/entrypoint.sh

Reply:
Removed this block in a simplifying commit below. (The original version of the script is still in the base image.)